docs/LV_client/tools/mono_logger.md

```diff
@@ -0,0 +1,55 @@
+# mono_logger.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_logger.png"/>
+</p>
+
+### Description
+
+This VI creates text files or binary files depending on the type of sensor.    
+**Text file:**        
+    - State sensor    
+    - Ultrasonic    
+    - Collision    
+    - Radar    
+ 
+**Binary files:**    
+    - IMU    
+    - GPS    
+    - RPM    
+    - LiDAR    
+    - Semantic LiDAR    
+    - Camera 
+
+It also saves a text file for the camera annotation if the user set to True the annotation flag.  
```

Comment:
Might want to reword this sentence:  "It also saves a text file for the camera annotation if the user sets the annotation flag to True."